hwdb.d/60-sensor.hwdb

```diff
@@ -262,7 +262,7 @@ sensor:modalias:acpi:*KIOX000A*:dmi:*svn*CytrixTechnology:*pn*Complex11t*
 #########################################
 # Dell
 #########################################
-sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:pnVostro5581:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*
```

Comment:
Can we be sure that this sensor is only used in the base (for all Dell products)?

Reply:
Sure, I'll request Dell's PM to confirm it.

Reply:
@keszybz 

with @superm1 confirmation, I think the rule can apply to all Dell platforms, please let me know if you have other concern.

Thanks,